src/js/components/App.tsx

```diff
@@ -36,9 +36,16 @@ const App = (props: AppSettings | any): JSX.Element => {
     //Determine which resources we are reading from
     //Read our local resources.js file And any external library resources (which are prioritized)
     dispatch(overwriteSettings({ ...cameroon, ...props }));
-    //Send that to our redux appSettings overwriting whatever is there
+    //Check URL for language param which comes in after user shares the application.
+    const langFromURL = new URL(window.location.href).searchParams.get('lang');
+    if (langFromURL) {
```

Comment:
@vaidotasp when I set the language and load the updated shareable URL, it looks the reloaded app doesn't update the language

Reply:
Thanks that should have been fixed with the latest commit! Re-test when you get a chance. Just heads up we still have issues with basemaps so english version may not load correctly, but at least it should listen properly to the URL hash.